### PR TITLE
Add entry for ember-query-params-service

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Self promotion. idk :)

### CODE REVIEW

File: source/2019-06-01-the-ember-times-issue-101.md [hunk 1]
```diff
@@ -55,12 +55,9 @@ We are excited to announce that [@rwwagner90](https://github.com/rwwagner90) has
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [🧪 Preview Version of ember-query-params-service](https://github.com/NullVoxPopuli/ember-query-params-service)
```
Comment (ijlee2) [inline source/2019-06-01-the-ember-times-issue-101.md]:
```suggestion
## [Preview Version of ember-query-params-service 🧪](https://github.com/NullVoxPopuli/ember-query-params-service)
```

File: source/2019-06-01-the-ember-times-issue-101.md [hunk 2]
```diff
@@ -55,12 +55,9 @@ We are excited to announce that [@rwwagner90](https://github.com/rwwagner90) has
...
+This is an experiment to see if using decorators and a separate service to manage query params is a feasible alternative
+to the existing query params implementation. We all **know** that query params in ember need to be fixed, and this is just one possible solution. It's a **very early** preview and is expected to have bugs. Should you choose to go on an adventure with this, please feel free to report weird or unexpected behavior. There will be an accompanying RFC to follow soon. 😉
```
Comment (ijlee2) [inline source/2019-06-01-the-ember-times-issue-101.md]: I might suggest removing the We all know part, to remain neutral. :) Would that be okay with you?
```suggestion
to the existing query params implementation. It's a **very early** preview and is expected to have bugs. Should you choose to go on an adventure with this, please feel free to report weird, unexpected behavior. There will be an accompanying RFC to follow soon. 😉
```

Comment (NullVoxPopuli) [reply source/2019-06-01-the-ember-times-issue-101.md]: Can do!

**Review by ijlee2 — APPROVED** [review]
Thanks for writing, I hope you will continue to!

**Pull request merged**